test/integration/local/processing/confirm-payment-request-number.test.js

```diff
@@ -0,0 +1,105 @@
+const db = require('../../../../app/data')
+
```

Comment:
Remove the space? Or keep them separated?

Reply:
Separated them so the focus of the test is separate